src/Http/Controllers/V1/Shop/Customer/AddressController.php

```diff
@@ -42,7 +42,7 @@ public function index(Request $request)
         $addresses = $customer->addresses()->get();
 
         return response([
-            'data' => CustomerAddressResource::collection($addresses),
+            'data' => !$addresses->isEmpty() ? CustomerAddressResource::collection($addresses) : __('rest-api::app.customers.no-address'),
         ]);
```

Comment:
For consistency, this can't be converted into a string. If data is not found then it should be a blank array. Rest if a third party wants to use our API then can simply check the length of the array.

Reply:
You can simply add a message key, but then this should be consistent also. If we provide a message key then it should be provided in the whole app as well.